example_dropin_environments/julia_mlj/dr_requirements.txt

```diff
@@ -1,3 +1,3 @@
 pandas==1.0.5
-pyarrow==0.14.1,<=10.0.1
+pyarrow==0.14.1,<=14.0.1
```

Comment:
@yakov-g I just found that our trviy checks failed on pyarrow still 
https://jenkins2.devinfra.drdev.io/job/CodeExperience-addons/job/notebooks-jenkins_trivy_groovy/257/artifact/reports/trivy_kernel-python-3.9-gpu.html

I apologize for not taking a closer look at this PR (I am _relatively_ unfamiliar with this repo.)
but it looks like we need to update multiple other places ... ??

https://github.com/search?q=repo%3Adatarobot%2Fdatarobot-user-models+pyarrow%3E%3D0.14.1%2C%3C%3D10.0.1&type=code

<img width="1095" alt="Screenshot 2023-11-15 at 9 33 38 AM" src="https://github.com/datarobot/datarobot-user-models/assets/1099971/ad4bf7b7-36b3-41c8-95e5-9418671c935e">